src/sagemaker/estimator.py

```diff
@@ -865,7 +891,7 @@ def _prepare_for_training(self, job_name=None):
         self._hyperparameters[JOB_NAME_PARAM_NAME] = self._current_job_name
         self._hyperparameters[SAGEMAKER_REGION_PARAM_NAME] = self.sagemaker_session.boto_region_name
 
-    def _stage_user_code_in_s3(self):
+    def _stage_user_code_in_s3(self, tar=True, s3_location='source'):
```

Comment:
source is not the location but the file name: source.tar.gz. Should we refactor this do reflect that?

Or we can keep the same name convention `source.tar.gz` for network isolation as well.

Reply:
I think 'source' is the location. The file name, `sourcedir.tar.gz` is specified here: https://github.com/aws/sagemaker-python-sdk/blob/master/src/sagemaker/fw_utils.py#L181.  But since we're going to tar everything we upload, we should just keep the same name convention.